packages/frontend/src/components/template/AdminLayout.tsx

```diff
@@ -39,7 +39,7 @@ export const AdminLayout = (props: IProps) => {
   return (
     <>
       <div id="adminlayout">
-      <div className="navbar navbar-dark bg-primary flex-md-nowrap p-0">
+      <div className="navbar header flex-md-nowrap">
```

Comment:
Is it related to the new menu add by @verdonarthur  ??

Reply:
Nope !